src/Tickets/Emails/Email/Purchase_Receipt.php

```diff
@@ -110,6 +110,24 @@ public function get_default_subject(): string {
 		return esc_html__( 'Your purchase receipt for #{order_number}', 'event-tickets' );
 	}
 
+	/**
```

Comment:
We already [have this method ](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Emails/Email_Abstract.php#L657)in the `Email_Abstract` class. We can potentially add the `apply_filters` there.